sig/environment_walker.rbs

```diff
@@ -0,0 +1,39 @@
+module RBS
+  class EnvironmentWalker
+    class InstanceNode
+      attr_reader type_name: TypeName
+      def initialize: (type_name: TypeName) -> void
+    end
+
+    class SingletonNode
+      attr_reader type_name: TypeName
+      def initialize: (type_name: TypeName) -> void
+    end
+
+    class TypeNameNode
+      attr_reader type_name: TypeName
+      def initialize: (type_name: TypeName) -> void
+    end
+
+    attr_reader env: Environment
+    attr_reader only_ancestors: bool
+    attr_reader builder: DefinitionBuilder
+
+    def initialize: (env: Environment) -> void
+
+    def only_ancestors!: (?bool only) -> self
```

Comment:
> __[Goodcheck]__ Prefer `boolish` over `bool` for method arguments and block return values ([view](https://sider.review/gh/repos/174796313/pulls/608#issue-5399461))
>
> <details>
>
> See the doc below:
> https://github.com/ruby/rbs/blob/78d04a2db0f1c4925d2b13c2939868edf9465d6c/docs/syntax.md#bool-or-boolish
>
> | Rule |
> | ---- |
> | `rbs.prefer_boolish` |
>
> You can [close this issue](https://sider.review/gh/repos/174796313/pulls/608#issue-5399461) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

I closed this issue, because:
- When you strictly want `true | false`.